src/datamodel/test.rs

```diff
@@ -0,0 +1,10 @@
+use crate::datamodel::character::*;
+
+
+#[test]
+fn test_minion() {
```

Comment:
The naming should be more indicative of the test, so that we get more information when a test case fails. I suggest something like `test_minions_of_mordred_can_see_each_other`.

Reply:
this is outdated, I've already changed this test's name